index.html

```diff
@@ -28,40 +28,65 @@ <h1>Portfolio</h1>
         <div class="portfolio-item">
             <a href="https://github.com/FelixLetheren/mayden-academy-logo"><img class="portfolio-img"
                                                                                 src="images/mayden-logo.png"
-                                                                                alt="Mayden Academy Logo"></a>
-            <a href="https://github.com/FelixLetheren/mayden-academy-logo"><h3>Mayden Academy logo</h3></a>
+                                                                                alt="Mayden Academy Logo">
+                <div class="portfolio-cover">
+                    <h2 class="portfolio-cover-link">GitHub link</h2>
+                </div>
+            </a>
+            <a href="https://github.com/FelixLetheren/mayden-academy-logo"><h3>Mayden Academy
```

Comment:
Open these links in a new tab for consistent behaviour

Reply:
Boom, done